app/models/Scenario.scala

```diff
@@ -28,6 +30,28 @@ case object Category2 extends Scenario
 
 object Scenario {
 
+  def getRedirectScenarios(categorisationInfo: CategorisationInfo): Scenario = {
+    val hasCategoryAssessments: Boolean =
+      categorisationInfo.categoryAssessments.length >= 1
```

Comment:
Is categoryAssessments a List? There's probably a scala native method for that, .nonEmpty or something?

Reply:
yep